phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -11,6 +11,39 @@
 <!-- BEGIN quote_username_open --><blockquote><div><cite>{USERNAME} {L_WROTE}{L_COLON}</cite><!-- END quote_username_open -->
 <!-- BEGIN quote_open --><blockquote class="uncited"><div><!-- END quote_open -->
 <!-- BEGIN quote_close --></div></blockquote><!-- END quote_close -->
+<!-- BEGIN quote_extended -->
+<blockquote>
+	<xsl:if test="not(@author)">
+		<xsl:attribute name="class">uncited</xsl:attribute>
+	</xsl:if>
+	<div>
+		<xsl:if test="@author">
+			<cite>
+				<xsl:if test="@date"><xsl:value-of select="@date"/> </xsl:if>
+				<xsl:choose>
+					<xsl:when test="@url">
+						<a href="{@url}" class="postlink"><xsl:value-of select="@author"/></a>
+					</xsl:when>
+					<xsl:when test="@profile_url">
+						<a href="{@profile_url}"><xsl:value-of select="@author"/></a>
+					</xsl:when>
+					<xsl:otherwise>
+						<xsl:value-of select="@author"/>
+					</xsl:otherwise>
+				</xsl:choose>
+				<xsl:text> </xsl:text>
+				<xsl:value-of select="$L_WROTE"/>
+				<xsl:value-of select="$L_COLON"/>
+				<xsl:if test="@post_url">
+					<xsl:text> </xsl:text>
+					<a href="{@post_url}" onclick="if(document.getElementById(this.hash.substr(1)))this.href=this.hash">&#8593;</a>
```

Comment:
Please do place this script's logic in one of the relevant external js files. Instead, use a relevant `data-*` attribute and use a selector query (E.g. `.querySelectorAll()` or `jquery()`) to apply the script on.
Anyway, IMO this is fine to stay as a script. I don't think there's a viable (and decoupled correct) way to do this in a different way. And this is just an extra QOL feature for the users.